### ISSUE TITLE
I am happy to contribute to this repo

### ISSUE THREAD

**Issue by herou** [zcash/zcash-android-wallet-sdk#183]
Hello,

Sorry for bothering you guys with this issue. My name is Eljo Prifti and I am an android developer who likes to contribute to this awesome repo.

Do you have any documentation and issue so I can start?

I hope to have your support.

**Comment by gmale** [comment]
Hi Herou! Sorry for delay, I was away for nearly a week due to the US Thanksgiving Holiday. I'm happy to accept any level of help at any time because I'm an Android team of 1, which is never easy.

Depending on what you're interested in, there are a lot of places to start. As far as documentation, the [main readme](https://github.com/zcash/zcash-android-wallet-sdk) is a decent source of information but it's probably easier to jump directly into the code by looking at the [demo app](https://github.com/zcash/zcash-android-wallet-sdk/tree/master/samples/demo-app). It's fairly up-to-date. Try getting that up and running.

In general, the most helpful things would be "tech debt" -- because that type of work is most likely to get skipped in favor of new features and bug fixes. A good warm up would be clearing some of the build warnings. 

For instance, there are a ton of warnings that say
> This declaration is experimental

and I'm pretty sure most of them could be cleared up by setting `freeCompilerArgs` like I did on the [ECC Wallet app](https://github.com/zcash/zcash-android-wallet/blob/e188c793a744c91b525f04ff75d54f08ac8b5100/app/build.gradle#L86-L91). I created a [github issue](https://github.com/zcash/zcash-android-wallet-sdk/issues/184) to track this, since it might be a good starting point.
